operator/src/main/java/oracle/kubernetes/operator/Main.java

```diff
@@ -228,14 +228,23 @@ private static void completeBegin() {
     }
   }
 
-  private static void stopNamespace(String ns, boolean remove) {
-    processor.stopNamespace(ns);
-    AtomicBoolean stopping =
-        remove ? isNamespaceStopping.remove(ns) : isNamespaceStopping.get(ns);
+  private static void stopNamespace(String ns, boolean inTargetNamespaceList) {
+    AtomicBoolean isNamespaceStopping = isNamespaceStopping(ns);
 
-    if (stopping != null) {
-      stopping.set(true);
+    // Remove if namespace not in targetNamespace list
+    if (!inTargetNamespaceList) {
+      Main.isNamespaceStopping.remove(ns);
```

Comment:
Should we rename `isNamespaceStopping` field in Main to something else, such as `isNamespaceStoppingMap` to avoid having same name?

Reply:
The name is really confusing. For example, we pass the keys from that map to the rest server:```

    startRestServer(principal, isNamespaceStopping.keySet())
```

when what we clearly mean is the list of all namespaces currently known. The problem is that it is a map of known namespace names to an atomic boolean which tells us if we are stopping that namespace, and there really isn't a good name for that. What that really suggests is that we ought to rethink the map itself, but that's a bit further off, I think.

For now, I would suggest dropping the "is." The usual convention is to use the "is" prefix for methods returning booleans, but not for the corresponding fields.

Reply:
Yes, changed field in Main to `isNamespaceStoppingMap`.

Reply:
Dropped the 'is.' from the 'isNamespaceStoppingMap' field in Main and renamed to 'namespaceStoppingMap' as suggested by Russ.